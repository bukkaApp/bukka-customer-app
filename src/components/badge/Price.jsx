import React from 'react';
import PropTypes from 'prop-types';

import './price.scss';

const Price = ({ price, classNames }) => (
  <h4 className={classNames}>â‚¦{price}.00</h4>
);

export default Price;

Price.defaultProps = {
  classNames: 'price-badge',
};

Price.propTypes = {
  price: PropTypes.oneOfType([
    PropTypes.string, PropTypes.number
  ]).isRequired,
  classNames: PropTypes.string,
};
