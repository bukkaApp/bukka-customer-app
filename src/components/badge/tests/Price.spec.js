import React from 'react';
import Price from '../Price';

describe('Price Component', () => {
  const wrapper = shallow(<Price price={17} />);

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
