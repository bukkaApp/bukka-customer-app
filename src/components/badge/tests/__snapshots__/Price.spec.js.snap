// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Price Component renders properly 1`] = `ShallowWrapper {}`;
